tests/auth_tests/test_models.py

```diff
@@ -155,6 +155,18 @@ def test_make_random_password(self):
         for char in password:
             self.assertIn(char, allowed_chars)
 
+    def test_create_user_no_email(self):
```

Comment:
This is already tested in `auth_tests.test_basic.BasicTestCase.test_user_no_email()` added in f54135fa4dc9fd45d86ea2944191d58e00725779. (So no need for it again.)

Reply:
Nice. I'll delete it.